docusaurus/docs/iOS/uikit/uikit-overview.md

```diff
@@ -0,0 +1,123 @@
+---
+title: UIKit Overview
+slug: /uikit
+---
+
+Building on top of the the Stream Chat API, the Stream Chat iOS component libraries include everything you need to build feature-rich and high-functioning chat user experiences out of the box.
+
+We have a component library available for both UIKit. The library includes an extensive set of performant and customizable UI components which allow you to get started quickly with little to no plumbing required. The libraries supports:
+
+- Rich media messages
+- Reactions
+- Threads and quoted replies
+- Text input commands (ex: Giphy and @mentions)
+- Image and file uploads
+- Video playback
+- Read state and typing indicators
+- Channel and message lists
+- Push (APN or Firebase)
+- Offline storage
+- OSX
+
+### Dependencies 
+
+This SDK tries to keep the list of external dependencies to a minimum, these are the dependencies currently used:
+
+#### StreamChatUI
+
+- [Nuke](https://github.com/kean/Nuke) for loading images  
+- [SwiftyGif](https://github.com/kirualex/SwiftyGif) for high performance GIF rendering
+- StreamChat the low-level client to Stream Chat API
+
+#### StreamChat
+
+- [Starscream](https://github.com/daltoniam/Starscream) to handle WebSocket connections
```

Comment:
This is duplicated. Shouldn't we remove this section from the Overview.md and only put this is here on the UIKit version?